docs/TestCoverage.md

```diff
@@ -4663,6 +4663,23 @@ expect(node, inputs=[data_0, data_1], outputs=[result],
        name='test_max_two_inputs')
 ```
 
+</details>
+<details>
+<summary>max_int8</summary>
+
+```python
+data_0 = np.array([ 3, 2,-1]).astype(np.int8)
```

Comment:
May we have one test for each added type?

Reply:
I didn't do that because a [previous comment](https://github.com/onnx/onnx/pull/2532#issuecomment-571787425) on a different PR said there were too many tests, but I would very happily add one for each type.

Reply:
I don't feel that comment is valid. Any code path not tested can be broken. :)